modules/navigation.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please remember to remove every linter rule that has been disabled and check for linter errors. You can also install linter checks locally using `npm` and the instructions provided by Microverse in the[ linters repository.](https://github.com/microverseinc/linters-config/tree/master/javascript)